.document

```diff
@@ -0,0 +1,3 @@
+LICENSE.txt
+README.md
+lib/
```

Comment:
Does it make sense to include `lib/` here?
I couldn't find any docs about a `.document` file for RDoc.

Reply:
`lib/timeout.rb` itself.
If no `.document` file, all files under the directory will be read.